execution_workflows/APA-Scan/workflow/rules/[SUBWORKFLOW].smk

```diff
@@ -0,0 +1 @@
+# Place any additional SUBWORKFLOWS here.
```

Comment:
To keep things clean, delete any template folders/files you don't need. Applies to more files further down as well